### DISCUSSION TITLE
Shutdown after activation function Power on/off button

### DISCUSSION THREAD

**Discussion by vigirex** [rern/rAudio#1590] [Fixed Issues]
Hello,
i've tried to activate in rAudio menu the function "Power on/off button" (without push button connected and also with button with no difference) and the board will be shutdown. When i've press the button (on 5 PIN) the Pi has started boot but after sometime already off. 
I've write new img of OS in SD card for use again.
Is it a bug? 
Is possible to solve without this inside function and made a script directly in Linux (python, bash, ecc)?
Thanks for help

**Comment by R101G** [comment]
The current 20240127 update fixes this. The previous update's default settings caused a shutdown after boot.
